README.md

```diff
@@ -6,79 +6,147 @@
+
+<div align="center">
 
-### _About_
+<img src="./docs/images/eva-process.gif" width="800">
 
-`eva` is [kaiko.ai](https://kaiko.ai/)'s evaluation framework for oncology foundation models (FMs). Check out the documentation (LINK TO BE ADDED) for more information.
+</div>
 
+### Highlights:
+- Easy and reliable benchmark of Oncology FMs
+- Automatic embedding inference and evaluation of a downstream task
+- Native support of popular medical [datasets](https://kaiko-ai.github.io/eva/dev/datasets/) and models
+- Produce statistics over multiple evaluation fits and multiple metrics
 
 ## Installation
 
-*Note: this section will be revised for the public package when publishing eva*
+Simple installation from PyPI:
+```sh
+# to install the core version only
+pip install kaiko-eva
 
-- Create and activate a virtual environment with Python 3.10+
+# to install the expanded `vision` version
+pip install 'kaiko-eva[vision]'
 
-- Install *eva* and the *eva-vision* package with:
+# to install everything
+pip install 'kaiko-eva[all]'
+```
 
+To install the latest version of the `main` branch:
+```sh
+pip install "kaiko-eva[vision] @ git+https://github.com/kaiko-ai/eva.git"
 ```
-pip install 'kaiko-eva[vision]'
+
+You can verify that the installation was successful by executing:
+```sh
+eva --version
 ```
 
-- To be able to use the existing configs, download them from the [*eva* GitHub repo](https://github.com/kaiko-ai/eva/tree/main) and move them to directory where you installed *eva*. 
+## How To Use
 
-### Run *eva*
+_eva_ can be used directly from the terminal as a CLI tool as follows:
+```sh
+eva {fit,predict,predict_fit} --config url/or/path/to/the/config.yaml 
+```
 
-Now you can run a complete *eva* workflow, for example with:
+For example, to perform a downstream evaluation of DINO ViT-S/16 on the BACH dataset with linear probing, by first infer the embeddings and perform 5 sequential fits, execute:
+```sh
+eva predict_fit --config https://raw.githubusercontent.com/kaiko-ai/eva/main/configs/vision/dino_vit/offline/bach.yaml
 ```
-eva fit --config configs/vision/dino_vit/online/bach.yaml 
+
+> [!NOTE] 
+> All the datasets the support automatic download in the repo have by default the option to `false` and thus you manually have to set the argument to `true`, that is `download=true`.
+
+
+To view all the possibles, execute:
+```sh
+eva --help
 ```
-This will:
 
- - Download and extract the dataset, if it has not been downloaded before.
- - Fit a model consisting of the frozen FM-backbone and a classification head on the train split.
- - Evaluate the trained model on the validation split and report the results.
+For more information, please refer to the [documentation](https://kaiko-ai.github.io/eva/dev/user-guide/tutorials/offline_vs_online/) and [tutorials](https://kaiko-ai.github.io/eva/dev/user-guide/advanced/replicate_evaluations/).
+
+## Benchmarks
+
+In this section you will find model benchmarks which were generated with _eva_.
```

Comment:
Maybe we could provide a link to the page in documentation here which shows how to reproduce this?

Reply:
good idea, maybe we need a dedicated  tab for that 🤔